content/spin/v2/quickstart.md

```diff
@@ -282,21 +279,23 @@ The additional `spin.toml` file is the manifest file, which tells Spin what even
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
+spin_manifest_version = "2"
 authors = ["You <you@yourself.net>"]
 description = "My first Rust Spin application"
 name = "hello_rust"
 trigger = { type = "http", base = "/" }
```

Comment:
This line needs to be removed (in favor of the `[[trigger.http]]` config added below.)